.github/workflows/workflow.yml

```
@@ -1,6 +1,7 @@
 name: Node CI
 
-on: [push]
+on:
+  push: 
```

il me semble que les `:` sont en trop.

```suggestion
on: push
```

`push` étant un event, il est utilisable au format liste comme au format unitaire.